src/sync.d

```diff
@@ -1757,7 +1757,7 @@ final class SyncEngine
 		
 		string site_id;
 		string drive_id;
-		JSONValue siteQuery = onedrive.o365SiteSearch(encodeComponent(o365SharedLibraryName));
+		JSONValue siteQuery = onedrive.o365SiteSearch(o365SharedLibraryName);
 		
```

Comment:
This needs to be fixed

Reply:
Thats weird .. I made the branch locally off master ... fixing ..